test/remoting/issue4007-reload-lost-app-window/test.py

```diff
@@ -0,0 +1,28 @@
+import time
+import os
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+chrome_options = Options()
+chrome_options.add_argument("nwapp=" + os.path.dirname(os.path.abspath(__file__)))
+
+driver = webdriver.Chrome(executable_path=os.environ['CHROMEDRIVER'], chrome_options=chrome_options)
+time.sleep(1)
+try:
+    print driver.current_url
+    result = driver.find_element_by_id('result').get_attribute('innerHTML')
+    print result
+    assert("success" in result)
+    driver.find_element_by_id('reloadapp').click()
+    # wait 10s for reload complete
+    # on Linux, the reload is quite slow
+    time.sleep(10)
```

Comment:
Is there any better way to do this rather than simply sleeping for random 10s? e.g. some blocking function call or event returns when the reload completes.